java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -2513,7 +2517,7 @@ else if (sa.getStatus().equals(ActionFactory.STATUS_QUEUED)) {
                             LOG.info("Scheduling a package profile update");
 
                             try {
-                                ActionManager.schedulePackageRefresh(minion.getOrg(), minion);
+                                ActionManager.schedulePackageRefresh(Optional.of(action.getSchedulerUser()), minion);
```

Comment:
same as above

Reply:
You're probably right. I changed them to `ofNullable(...)`